zone/attack.cpp

```diff
@@ -2507,8 +2507,9 @@ bool NPC::Death(Mob* killer_mob, int64 damage, uint16 spell, EQ::skills::SkillTy
 	d->spell_id     = UINT32_MAX;
 	d->attack_skill = SkillDamageTypes[attack_skill];
 	d->damage       = damage;
+	d->corpseid		= GetID();
 
-	app->priority = 6;
+	app->priority = 1;
```

Comment:
In my testing I noted that the priority was negatively influencing the order that the packets arrived at the client.  I made the change to ensure that they arrived in the correct order so that the rename would take effect.